src/AzureIoTHub.Portal/Client/Program.cs

```diff
@@ -65,7 +65,7 @@ private static async Task ConfigureOidc(WebAssemblyHostBuilder builder)
                 options.ProviderOptions.ClientId = settings.ClientId;
 
                 options.ProviderOptions.DefaultScopes.Add(settings.Scope);
-                options.ProviderOptions.ResponseType = "id_token token";
+                options.ProviderOptions.ResponseType = "code";
```

Comment:
Replace "code" by OpenIdConnectResponseType.Code

Reply:
This suppose to install another NuGet package ``Microsoft.IdentityModel.Protocols.OpenIdConnect`` just for the string convention, I would prefer to hard code that ...